linux_os/guide/system/software/updating/security_patches_up_to_date/rule.yml

```diff
@@ -33,7 +33,11 @@ rationale: |-
     users may take advantage of weaknesses in the unpatched software. The
     lack of prompt attention to patching could result in a system compromise.
 
-severity: high
+severity: {{% if product == "ol7" -%}}
+medium
+{{%- else -%}}
+high
+{{%- endif %}}
```

Comment:
I wouldn't make this change at all. This severity has to be `high`.

Reply:
@redhatrises thanks for your guidance.
I don't mind to have it "high" and was confused by "medium" for this rule as well. However latter is in use in DISAs OL7 STIG v1r1. Do you think this was a mistake on their side and expected to be fixed in the next STIG update?